local/README.md

```diff
@@ -25,30 +23,36 @@ These playbooks will:
           $ python -c "import ansible;print(ansible.__version__)"
           2.2.2.0
   * Install python dependencies
-     * `pip install six`
+        $ pip install six
 
 ### Execute
-  * `cd local`
-  * Edit the variables file `local/common_vars`
-    * Update:
-      * CLUSTER_IP if your installation of Docker is not using the default bridge of `docker0`
-  * `./run_setup_local.sh`
-    * Sets up OpenShift
-  * In Web Browser
+  * Navigate to the `local` folder
+        $ cd catasb/local
+  * Edit the variables file `common_vars`
+    * Note: `CLUSTER_IP` assumes that the default bridge of `docker0` is being used
+    * Optional: Edit the variables file `my_vars`
+          $ more my_vars
+          export DOCKERHUB_USER_NAME="docker_user_name"
+          export DOCKERHUB_USER_PASSWORD="my_password"
+          export DOCKERHUB_ORG_NAME="my_org"
+
+  * Run the setup script
+          $ ./run_setup_local.sh
+  * Open a Web Browser
     * Visit: `https://apiserver-service-catalog.CLUSTERIP.nip.io`
-      * Accept the certificate
-      * You will see some text on the screen, ignore this and proceed to the main openshift URL next
-       * Point of this step is just to accept the SSL cert for the apiserver-service-catalog endpoint
-    * Visit: `https://CLUSTERIP.nip.io:8443`
+      * Accept the SSL certificate for the apiserver-service-catalog endpoint
+      * Ignore the text that appears and proceed to the main OpenShift URL next
+      * Note: must accept the new SSL cert, each time you reset your OpenShift environment
+    * Visit: `https://<CLUSTERIP>.nip.io:8443`
+      * The <CLUSTERIP> is the same as the one you set in `common_vars`
 
 ### Cleanup
 
-To terminate the local instance run the below
-  * `oc cluster down`
+* To terminate the local instance run the below
+      $ oc cluster down
 
-To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
-  * `cd local`
-  * `./reset_environment.sh`
+* To reset the environment to a clean instance of origin with ASB and Service Catalog run the below
+      $ ./reset_environment.sh
```

Comment:
Again, these commands get butted to the end of the sentences and you can't really see they are commands that need executing.